### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by telatin** [phac-nml/irida-linker#5]
The short usage provided by the script:
```
Usage:
    ngsArchiveLinker.pl -b <API URL> -p <projectId> -d <outputDirectory> [-s
    <sampleId> ...]
```

suggests to use **-d** to specify the output directory, while it should be **-o**, as **-d** triggers the download (instead of symlink), instead

**Comment by apetkau** [comment]
Thanks @telatin. I can see there's a commit made by you that fixes this issue. Did you want to submit this as a pull request and we can merge it in?

**Comment by telatin** [comment]
Just tried. Thanks!

**Comment by tom114** [comment]
Fixed in #6
